
Fix build with abseil-cpp-20240722.0 which no longer implicitly
includes absl/log/vlog_is_on.h

See: https://github.com/abseil/abseil-cpp/releases/tag/20240722.0
Bug: https://bugs.gentoo.org/939015

--- grpc-1.65.1/src/core/util/log.cc~	2024-07-17 00:53:49.000000000 +0200
+++ grpc-1.65.1/src/core/util/log.cc	2024-09-04 09:28:18.494476262 +0200
@@ -19,6 +19,7 @@
 #include <grpc/support/port_platform.h>
 
 #include "absl/log/log.h"
+#include "absl/log/vlog_is_on.h"
 
 #include <stdio.h>
 #include <string.h>
